taca/utils/undetermined.py

```diff
@@ -171,8 +174,13 @@ def check_index_freq(run, lane, freq_tresh):
     :returns: True if the checks passes, False otherwise
     """
     barcodes={}
-    if os.path.exists(os.path.join(run, dmux_folder,'index_count_L{}.tsv'.format(lane))):
+    index_count_file=os.path.join(run, dmux_folder,'index_count_L{}.tsv'.format(lane))
+    if os.path.exists(index_count_file) :
         logger.info("Found index count for lane {}.".format(lane))
+        if os.stat(index_count_file).st_size !=0:
```

Comment:
I do not get this, if the file exists (i.e., it has been already created) and the size of it is larger than 0 (i.e., at least one byte is written), you return False... this will return false all the time after the .tsv has been computed.... or not?
Plus: this set up does not solve the race condition. There still might be situation (really unfortunate) in which you operate on files that are not yet done.
@Galithil my version should solve these problems and improve (stabilise) the QC, can you take away these changes so they will not conflict with mine?